instance/migrations/0128_fill_dns_records_updated_field.py

```diff
@@ -0,0 +1,37 @@
+# Generated by Django 2.2.9 on 2020-04-15 19:50
```

Comment:
Can you merge these 2 migrations into one and rebase it on master? 
There are a few conflicting migrations.

Reply:
@giovannicimolin I already rebased on top of master yesterday and manually resolved migration dependency conflicts. Are you still running into issues with the migrations?
I did not merge the two migrations into a single one though. Would there be any benefit in doing that?

Reply:
>Would there be any benefit in doing that?

Not really, just having less migrations. :stuck_out_tongue_closed_eyes:
This is just a nit so it's fine to keep it as is.